esx_service/utils/auth_api.py

```diff
@@ -472,6 +472,12 @@ def _tenant_create(name, default_datastore, description="", vm_list=None, privil
         if error_info:
             return error_info, None
 
+        error_info = vmdk_utils.check_volumes_mounted(vms)
+        if error_info:
+            error_info.msg = "VMs already have volumes in use, cannot add VMs to vmgroup. " + error_info.msg
```

Comment:
`error_info.msg = "Cannot add VM to vmgroup " + error_info.msg`

to be consistent with other error message and error could be "VM not found"